pkg/agent/qrm-plugins/io/handlers/iocost/iocost_linux.go

```diff
@@ -265,6 +295,15 @@ func SetIOCost(conf *coreconfig.Configuration,
 		return
 	}
 
+	// Strict mode: checking wbt file.
+	if conf.IOCostStrictMode {
```

Comment:
will it be better to replace IOCostStrictMode with CheckWBTDisabled since it's only used here?

Reply:
Done.